app/scripts/modules/appengine/domain/IAppengineLoadBalancer.ts

```diff
@@ -1,6 +1,7 @@
 import {ILoadBalancer} from 'core/domain/index';
 
 export interface IAppengineLoadBalancer extends ILoadBalancer {
+  credentials?: string;
   split: IAppengineTrafficSplit;
   migrateTraffic: boolean;
```

Comment:
The `migrateTraffic` checkbox is now disabled in the cases where it would be invalid. From the help text, it should be clear to the user why this is so.